src/vcpkg/build.cpp

```diff
@@ -1527,13 +1529,26 @@ namespace vcpkg
             .append_raw(spec_name)
             .append_raw('\n');
         result.append(msgBuildTroubleshootingMessage2).append_raw('\n');
-        result.append_indent()
-            .append_fmt_raw("https://github.com/microsoft/vcpkg/issues/"
-                            "new?template=report-package-build-failure.md&title=[{}]+Build+error",
-                            spec_name)
-            .append_raw('\n');
-        result.append(msgBuildTroubleshootingMessage3, msg::package_name = spec_name).append_raw('\n');
-        result.append_raw(paths.get_toolver_diagnostics()).append_raw('\n');
+        if (issue_body.has_value())
+        {
+            auto path = issue_body.get()->generic_u8string();
+            result.append_indent().append_fmt_raw("https://github.com/microsoft/vcpkg/issues/"
+                                                  "new?title=[{}]+Build+error&body=Copy+issue+body+from+{}\n",
```

Comment:
In this URI why are the spaces in the title turned into `+`s but not spaces in the `Strings::percent_encode(path)` ?

Reply:
Because percent encoding does not allow to decode spaces as `+`, in general the URI RFC does not know the `+` encoding for spaces, but in the HTML spec they say the body of a form is [`application/x-www-form-urlencoded`](https://www.w3.org/TR/html401/interact/forms.html#h-17.13.4.1) and can be submitted [via "get"](https://www.w3.org/TR/html401/interact/forms.html#h-17.13.3.4). 

Because `+` is shorter than `%20` I used is here for the spaces.